Cargo.toml

```diff
@@ -22,3 +24,9 @@ env_logger = "0.7.1"
 [build-dependencies]
 bindgen = "0.52.0"
 pkg-config = "0.3.17"
+
+[features]
+docs = []
```

Comment:
Maybe we will also have to add some of thse dependencies as optional if they do not build on the docs.rs server.

Reply:
... hopefully they do